### ISSUE TITLE
Thumb arc feelings

### ISSUE THREAD

**Issue by algernon** [algernon/Model01-sketch#32]
I've been trying to keep an eye on how I use the thumb arc, which keys are easiest to reach, and so on.

So far, it looks like that the middle two buttons are the easiest to reach, the innermost one is harder to reach, but when I do, accuracy does not suffer. The outermost one is easier to reach than the innermost, but seems to have an accuracy penalty.

With this in mind, some of the keys may have to move. The base layer seems okayish (though a `Backspace` / `Shift` & `Enter` / `Space` swap may be worth an experiment), but on `_NAV`, the mouse buttons should perhaps be in the middle, surrounded by `PgUp` / `PgDn`.

Mind you, all of these may need more data...

**Comment by algernon** [comment]
Turns out, it's all a matter of hand positioning. I tried the `Space`/`Enter` swap, and it didn't work. Due to #34, the mouse buttons won't be in the middle, either. The `:` and `-` keys feel okay-ish, but they don't feel as good as they do on their ErgoDox location. So perhaps they should be put elsewhere, and some other symbol or key put on the arc.

`Control` / `Alt` may be a reasonable candidate, swapping them with `:`/`-`...
